wpilib/wpilib/joystick.py

```diff
@@ -37,15 +39,27 @@ class AxisType:
         kZ = 2
         kTwist = 3
         kThrottle = 4
+
+    class Axis:
```

Comment:
Ok. Why is Axis and AxisType identical?

Reply:
I have **zero** idea. I think just for deprecation stuff? That reminds me...